webapp/src/shared/components/RequireAuth/RequireAuth.css

```diff
@@ -0,0 +1,10 @@
+.require-auth {
+  min-height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
```

Comment:
Since we use these styles in all the pages, I think we could add it to the `body` in the `index.css` and remove it from all the pages (including this one). What do we think about this?